src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBroker.cs

```diff
@@ -124,35 +124,23 @@ private async Task InvokeIosBrokerAsync(Dictionary<string, string> brokerPayload
             {
                 _logger.Info(iOSBrokerConstants.BrokerPayloadContainsInstallUrl);
 
-                string url = brokerPayload[BrokerParameter.BrokerInstallUrl];
```

Comment:
@trwalke - I move this code higher up, as discussed. This parses an auth code URI and extracts the app link. The way you'll see it in the Android broker is:

``` BrokerPayload[BrokerParameter.BrokerInstallUrl]```

Android broker should start an activity to navigate to this url, if the key is present in that dictionary.